pypose/function/geometry.py

```diff
@@ -126,8 +126,14 @@ def reprojerr(points, pixels, intrinsics, extrinsics=None):
             The shape has to be (..., 3, 3).
         extrinsics (``LieTensor``, optional): The camera extrinsics.
             The shape has to be (..., 7). Default: ``None``.
+        reduction (``str``, optional): The reduction to apply on the output: ``"none"`` | ``"sum"`` | ``"norm"``
```

Comment:
Line exceeding 90 chars, see point 2 of this [guideline](https://github.com/pypose/pypose/wiki/Developer-Guideline).